File: UniversityAPI/Entities/DTO/StdRemoveVM.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Entities.DTO
{
    public class StdRemoveVM
    {
        public int StudentId { get; set; }
        public string StudentName { get; set; }

        public int ErrorCode { get; set; }
        public string ErrorMessage { get; set; }

        public List<StdCrsRemoveVM> Courses { get; set; }

        public List<StdAsmtRemoveVM> Assignments { get; set; }
    }
}

File: NgApp/src/app/models/stdRemoveVM.ts
import StdAsmtRemoveVM from "./stdAsmtRemoveVM";
import StdCrsRemoveVM from "./stdCrsRemoveVM";

export default class StdRemoveVM {
    studentId: number;
    studentName: string;
    errorCode: number;
    errorMessage: string;
    courses: StdCrsRemoveVM[];
    assignments: StdAsmtRemoveVM[];
}
